app/api/item_routes.py

```diff
@@ -23,3 +23,18 @@ def item(item_id):
         return abort(404)
 
     return item.to_dict()
+
+
+@item_routes.route("/category=<int:category_id>")
+def items_in_category(category_id):
```

Comment:
Filtering items by category should be included in the `/` route in line 8.
To expand on the current filtering we have, it might be better if we make the filtering a bit more dynamic and easier for us to work with.

If we refactor it to something like below, It should allow us to easily add more filters conditionally. You can build upon this to add your categories filtering.

```py
@item_routes.route("/")
def items():
    key = request.args.get("key")
    filters = []
    if key:
        filters.append(Item.name.ilike(f"%{key}%"))
    items = Item.query.filter(*filters).all()
    return {"items": [item.to_dict() for item in items]}
```